tier4_system_msgs/msg/EmergencyGoalsStamped.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time stamp
+geometry_msgs/Pose[] goals
```

Comment:
Is `EmergencyGoalsClearCommand` necessary? How about considering it as clear command if there are zero goals?

Reply:
I guess it's a matter of taste, but I think the clear command is clearer in intent than having the number of goals mean clear.
It is not a continuous increase or decrease in the number of goal poses, but a completely different behavior, so I think it is acceptable to separate the topics.

Reply:
OK. That make sense.